src/scripts/main.js

```
@@ -21,7 +21,32 @@ const food = {
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  if (!element) {
+    return;
+  }
```

The function checks `if (!element) return;` which prevents errors when `element` is falsy, but it doesn't verify that `element` is actually a DOM node. Consider tightening this guard so the function fails early and clearly if the caller passes a non-element (for example: `if (!(element instanceof Element)) return;`). This makes the contract explicit: the first parameter must be a DOM element (requirement: first argument is a DOM element container).